src/palindrom-network-channel.js

```diff
@@ -70,6 +70,18 @@ export default class PalindromNetworkChannel {
                 return useWebSocket;
             }
         });
+
+        if (pingIntervalS) {
+            const intervalMs = pingIntervalS * 1000;
+            this.heartbeat = new Heartbeat(
+                () => {this.send([]);},
+                this._handleConnectionError.bind(this),
+                intervalMs,
+                intervalMs
```

Comment:
I know this is copied from existing implementation, but I think it does not make sense to use the same value for timeout that is used for ping interval.

It seems better to add another configuration option for timeout or even hardcode it temporarily and make an issue to fix it in the future.

Reply:
I'd make an issue, to keep this only as refactoring.

Reply:
https://github.com/Palindrom/Palindrom/issues/254